misc/python/materialize/parallel_workload/database.py

```diff
@@ -35,7 +35,7 @@
 
 MAX_COLUMNS = 100
 MAX_INCLUDE_HEADERS = 5
-MAX_ROWS = 1000
+MAX_ROWS = 100
```

Comment:
Why did we have to decrease the rows by factor 10?

Reply:
Because previously we nearly never reached 1000 rows, but instead mostly had 0-10. Now the max is much more likely to be reached, and joins and some other queries easily baloon to dozens of GBs of memory. I'm not sure if this is even enough yet.